hosted/aks/p1/p1_provisioning_test.go

```diff
@@ -96,22 +100,22 @@ var _ = Describe("P1Provisioning", func() {
 		Expect(cluster.AKSStatus.UpstreamSpec.Tags).To(HaveKeyWithValue("empty-tag", ""))
 	})
 
-	XIt("should be able to create cluster with container monitoring enabled", func() {
-		// blocked by https://github.com/rancher/shepherd/issues/274
+	It("should be able to create cluster with container monitoring enabled", func() {
+		// Refer: https://github.com/rancher/shepherd/issues/274
 		testCaseID = 199
 		updateFunc := func(aksConfig *aks.ClusterConfig) {
 			aksConfig.Monitoring = pointer.Bool(true)
 		}
 		var err error
 		cluster, err = helper.CreateAKSHostedCluster(ctx.RancherAdminClient, clusterName, ctx.CloudCredID, k8sVersion, location, updateFunc)
 		Expect(err).To(BeNil())
+		Expect(*cluster.AKSConfig.Monitoring).To(BeTrue())
+
 		cluster, err = helpers.WaitUntilClusterIsReady(cluster, ctx.RancherAdminClient)
 		Expect(err).To(BeNil())
 
 		helpers.ClusterIsReadyChecks(cluster, ctx.RancherAdminClient, clusterName)
-
-		Expect(*cluster.AKSConfig.Monitoring).To(Equal(true))
-		Expect(*cluster.AKSStatus.UpstreamSpec.Monitoring).To(Equal(true))
```

Comment:
Do include this check as well

Reply:
It has been included, but I am checking for it after cluster creation, check L#112. I have simply refactored the checks.